tests/validation/tests/v3_api/test_multi_cluster_app.py

```diff
@@ -362,6 +347,70 @@ def test_multi_cluster_all_answer_override():
     client.delete(mcapp)
 
 
+@if_test_rbac
+@pytest.mark.parametrize("role", ["owner", "member", "read-only"])
```

Comment:
since `["owner", "member", "read-only"]` appears in multiple places, can we make it a variable?